kotlin/src/main/java/org/apache/fury/serializer/kotlin/KotlinSerializers.java

```diff
@@ -94,5 +100,48 @@ public static void registerSerializers(Fury fury) {
         resolver.registerSerializer(UIntArray.class, new UIntArraySerializer(fury));
         resolver.register(ULongArray.class);
         resolver.registerSerializer(ULongArray.class, new ULongArraySerializer(fury));
+
+        // Ranges and Progressions.
+        resolver.register(kotlin.ranges.CharRange.class);
+        resolver.register(kotlin.ranges.CharProgression.class);
+        resolver.register(kotlin.ranges.IntRange.class);
+        resolver.register(kotlin.ranges.IntProgression.class);
+        resolver.register(kotlin.ranges.LongRange.class);
+        resolver.register(kotlin.ranges.LongProgression.class);
+        resolver.register(kotlin.ranges.UIntRange.class);
+        resolver.register(kotlin.ranges.UIntProgression.class);
+        resolver.register(kotlin.ranges.ULongRange.class);
+        resolver.register(kotlin.ranges.ULongProgression.class);
+
+        // Built-in classes.
+        resolver.register(kotlin.Pair.class);
+        resolver.register(kotlin.Triple.class);
+        resolver.register(kotlin.Result.class);
+        resolver.register(Result.Failure.class);
+
+        // kotlin.random
+        resolver.register(KotlinToJavaClass.INSTANCE.getRandomDefaultClass());
+        resolver.register(KotlinToJavaClass.INSTANCE.getRandomInternalClass());
+        resolver.register(KotlinToJavaClass.INSTANCE.getRandomSerializedClass());
+
+        // kotlin.text
+        resolver.register(Regex.class);
+        resolver.register(KotlinToJavaClass.INSTANCE.getRegexSerializedClass());
+        resolver.register(RegexOption.class);
+        resolver.register(CharCategory.class);
+        resolver.register(CharDirectionality.class);
+        resolver.register(HexFormat.class);
+        resolver.register(MatchGroup.class);
+
+        // kotlin.time
+        resolver.register(DurationUnit.class);
+        resolver.register(Duration.class);
+        resolver.registerSerializer(Duration.class, new DurationSerializer(fury));
+        resolver.register(TestTimeSource.class);
```

Comment:
Do we need to register this test class by default?

Reply:
Hmm I was debating this, let's remove for now.